src/sql/workbench/browser/modelComponents/table.component.ts

```diff
@@ -325,6 +350,154 @@ export default class TableComponent extends ComponentBase implements IComponent,
 		}
 	}
 
+	private registerRowDetailPlugin(settings: azdata.RowDetailsOptions) {
+		const rowDetail = new RowDetailView({
+			cssClass: '_detail_selector',
+			process: (item) => {
+				(<any>rowDetail).onAsyncResponse.notify({
+					'itemDetail': item,
+				}, undefined, this);
+			},
+			useRowClick: true,
+			panelRows: settings.panelRows,
+			postTemplate: (itemDetail) => itemDetail.__detailsData__,
+			preTemplate: () => '',
+			loadOnce: true,
+			detailsHtml: settings.detailsHtml
+		});
+		this._rowDetail = rowDetail;
+		this._table.registerPlugin(<any>this._rowDetail);
+	}
+
+	private registerFilterPlugin() {
+		const filterPlugin = new HeaderFilter<Slick.SlickData>();
+		this._register(attachButtonStyler(filterPlugin, this.themeService));
+		this._filterPlugin = filterPlugin;
+		this._filterPlugin.onFilterApplied.subscribe((e, args) => {
```

Comment:
Make sure you're calling unsubscribe when this is destroyed. Something like https://github.com/microsoft/azuredatastudio/blob/main/src/sql/base/browser/ui/table/plugins/headerFilter.plugin.ts#L49

Reply:
The same issue was for rowDetailsPlugin as well. Fixed for both.